data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/s3/S3SourceConfig.java

```diff
@@ -24,6 +24,7 @@
 
 public class S3SourceConfig {
     static final Duration DEFAULT_BUFFER_TIMEOUT = Duration.ofSeconds(10);
+    static final Duration DEFAULT_NUMBER_OF_WORKERS = 5;
```

Comment:
I think we should leave this as `1` so that this change does not impact existing pipelines. I believe with large object sizes `1` is the ideal value also.